sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/Generated/Models/Activity.cs

```diff
@@ -28,7 +28,6 @@ public Activity(string name)
             DependsOn = new ChangeTrackingList<ActivityDependency>();
             UserProperties = new ChangeTrackingList<UserProperty>();
             AdditionalProperties = new ChangeTrackingDictionary<string, object>();
-            Type = "Activity";
```

Comment:
@wonner, do you have any concerns about removing the line that sets this value here?

Reply:
It is an [internal property](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/Generated/Models/CopySink.cs#L43), so customers would not be accessing it directly.  However,[ it is used for serialization](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/Generated/Models/CopySink.Serialization.cs#L23), so if we did not set the default value, the derived type might not be able to be deserialized correctly.

Reply:
The answer to this is that it's ok to remove these discriminator values from the base type because the derived type will always populate the Type value.  There is not a risk to serialization if this is removed because of a discriminator is not provided, deserialization falls through to the base class, for example here: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/Generated/Models/CopySink.Serialization.cs#L170